test/e2e/e2e_test.go

```diff
@@ -155,6 +157,13 @@ func TestPubSubStackDriverMetrics(t *testing.T) {
 	PubSubWithTargetTestImpl(t, packageToImageConfig, true /*assertMetrics */)
 }
 
+// TestPubSubWithBroker tests we can knock a knative service from a broker with PubSub channel.
```

Comment:
// TestPubSubWithBroker tests we can knock a Knative Service from a Broker with PubSub Channel.

Reply:
actually maybe we can call it `TestBrokerWithPubSubChannel`... `TestPubSubWithBroker` might get confusing as it might mean a PubSub source to a Broker.